specs/interop/predeploys.md

```diff
@@ -0,0 +1,243 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Predeploys
+
+Two new system level predeploys are introduced for managing cross chain messaging along with
+an update to the `L1Block` contract with additional functionality.
+
+## CrossL2Inbox
```

Comment:
Ok so if we end up shipping with the feature where you can pull in events from L1 then having this be named the `CrossL2Inbox` really only displays a subset of its functionality. It may be worth just renaming it `Inbox` for simplicity (or `Portal`) but also don't want to bikeshed on names too much.

h/t @annieke

Reply:
Either's fine with me! +1 on not bikeshedding, we can just make diagrams to show how contracts relate to one another (instead of matching names)